### PR TITLE
feat: Transform Plus button to Minus in case of Alt button is pressed

### PR DESCRIPTION
**Pull request by istarkov**
## Description

ref #3994

image

image

image

## Steps for reproduction

1. click button
2. expect xyz

## Code Review

- [ ] hi @kof, I need you to do
  - conceptual review (architecture, feature-correctness)
  - detailed review (read every line)
  - test it on preview

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 0000)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env` file

### CODE REVIEW

**Review by TrySound — APPROVED** [review]

**Comment by kof** [comment]
- [ ] may be not part of this pr but reporting it anyways, what do we do if templates has nothing inside?

image

**Comment by istarkov** [comment]
Havent did anything with empty for now. 
We just discussed that Content Block without Templates is just wrapper which makes internal elements visible in navigation panel

**Comment by kof** [comment]
- [ ] when no instances inside, click "-" and it will delete the content block itself, is this supposed to happen?

**Comment by istarkov** [comment]
Added to https://github.com/webstudio-is/webstudio/issues/3994

**Comment by istarkov** [comment]
> when no instances inside, click "-" and it will delete the content block itself, is this supposed to happen?

good catch, also tooltips should be different

**Comment by kof** [comment]
- [ ] lets make the button in destructive color when it shows deletion 
- [ ] lets use a trash icon

**Comment by kof** [comment]
- [ ] menu item height should be 24px in both cases and anywhere else if you see those being used

image

image

**Review by kof — APPROVED** [review]

File: apps/builder/app/builder/features/workspace/canvas-tools/outline/block-instance-outline.tsx [hunk 1]
```diff
@@ -0,0 +1,504 @@
...
+  Kbd,
+  Text,
+  iconButtonSize,
+  Grid,
+  DropdownMenuSeparator,
+  menuItemCss,
+} from "@webstudio-is/design-system";
+import { Outline } from "./outline";
+import { applyScale } from "./apply-scale";
+import { $scale } from "~/builder/shared/nano-states";
```
Comment (TrySound) [inline apps/builder/app/builder/features/workspace/canvas-tools/outline/block-instance-outline.tsx:33]: not builder specific anymore

File: apps/builder/app/canvas/shared/styles.ts [hunk 2]
```diff
@@ -138,65 +138,90 @@ const subscribeDesignModeHelperStyles = () => {
...
+  renderHelperStyles();
+
+  const requestIdleCallbackFn =
+    typeof requestIdleCallback !== "undefined"
+      ? requestIdleCallback
+      : requestAnimationFrame;
+  const cancelIdleCallbackFn =
+    typeof cancelIdleCallback !== "undefined"
+      ? cancelIdleCallback
+      : cancelAnimationFrame;
```
Comment (TrySound) [inline apps/builder/app/canvas/shared/styles.ts:211]:
````
const requestIdleCallbackFn = globalThis.requestIdleCallback ?? requestAnimationFrame
const cancelIdleCallbackFn = globalThis.cancelIdleCallback ?? cancelAnimationFrame
```

**Pull request merged**